app-center-services/src/main/java/org/exoplatform/appcenter/util/Utils.java

```diff
@@ -0,0 +1,18 @@
+package org.exoplatform.appcenter.util;
+
+import org.exoplatform.services.listener.ListenerService;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+
+public class Utils {
```

Comment:
No ned to add a whole class for a simple function call. please remove this class and include the call inside the service iyself.

Reply:
Is this still needed ?